zad6/app/models/product.rb

```diff
@@ -0,0 +1,9 @@
+class Product < ApplicationRecord
+  belongs_to :customer
```

Comment:
Ta linia nie pozwala na stworzenie produktu, który nie ma jeszcze przypisanego customera. Można to poprawić w następujący sposób:
```ruby
belongs_to :customer, optional: true
```